src/metrics.js

```diff
@@ -60,9 +58,6 @@ function sendFeatureMetric(name) {
     return;
   }
 
-  if (!Kite) {
-    Kite = require("./kite");
```

Comment:
@ryantberg I opted to remove these requires and import Kite instead since it looked like Kite.request doesn't rely on any properties specific to the Kite object. Correct me if this isn't a safe refactoring / doesn't preserve the intent here though.

Reply:
This seems safe/preferred to me. There's no real advantage to lazy-loading the module that I can see.